superset-frontend/src/SqlLab/components/SaveDatasetModal/index.tsx

```diff
@@ -61,7 +61,7 @@ export type ExploreQuery = QueryResponse & {
 };
 
 export interface ISimpleColumn {
-  name?: string | null;
+  column_name?: string | null;
```

Comment:
This is great to see this. IIRC there were quite a few places in the react codebase where we had to look up either column_name or name. I found one here: superset-frontend/packages/superset-ui-chart-controls/src/utils/getTemporalColumns.ts Should we do a sweep through and look for more?